scipy/stats/_continuous_distns.py

```diff
@@ -5959,6 +5959,46 @@ def _entropy(self):
 semicircular = semicircular_gen(a=-1.0, b=1.0, name="semicircular")
 
 
+class sine_gen(rv_continuous):
+    r"""A sine continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The sine distribution is useful for polar angles.
+    The probability density function for `sine` is:
+
+    .. math::
+
+        f(x) = \frac{1}{2} \sin(x)
```

Comment:
Probably simpler to write 0.5 instead of frac, there was a discussion to minimize LaTeX expressions some time ago